src/lib/core/say.h

```diff
@@ -172,16 +172,14 @@ struct log {
 	/* Application identifier used to group syslog messages. */
 	char *syslog_ident;
 	/**
-	 * Used to wake up the main logger thread from a eio thread.
+	 * Counter identifying number of threads executing log_rotate.
+	 * Protected by @mutex as it is accessed from different threads.
```

Comment:
Nit: `rotate_mutex`. Also, we don't usually use `@` to refer to variables and functions. Let's also elaborate the comment to `rotate_mutex` - "mutex for critical sections" sounds too vague.

Reply:
Done.